### ISSUE TITLE
[BUG]: Can't Login After Upgrade From 0.20.2

### ISSUE THREAD

**Issue by MrMYHuang** [activepieces/activepieces#4542]
Describe the bug
After more study for the same issue in https://github.com/activepieces/activepieces/issues/4528, I find the key point of the issue: the login password does not work anymore! I repeat the steps to reproduce the issue in the following:

I set up my activepieces in a docker-compose.yml file, like this:

```
    activepieces:
      image: activepieces/activepieces:0.20.2
      privileged: true
      ports:
        - 8892:80
      volumes:
        - ~/.activepieces:/root/.activepieces
      environment:
        - AP_QUEUE_MODE=MEMORY
        - AP_EXECUTION_MODE=SANDBOXED
        - AP_SIGN_UP_ENABLED=true
        - AP_DB_TYPE=SQLITE3
        - AP_TRIGGER_DEFAULT_POLL_INTERVAL=1
        - AP_CLOUD_AUTH_ENABLED=true
        - AP_WEBHOOK_TIMEOUT_SECONDS=300
        - AP_FRONTEND_URL=https://activepieces.example.com
      networks:
        - backend
 ```
After I update the activepieces version from 0.20.2 to a higher version (e.g., 0.21.0 or 0.25.1) in docker-compose.yml, then I execute docker-compose up -d, activepieces, it starts successfully. But I couldn't login my account, the webpage said "Invalid email or password."

Actually, on my one Ubuntu Linux ARM64 machine, I tried to sign up another new account with updated activepieces, and copied its encrypted password to my old account's password, for example:
```
sudo sqlite3 ~/.activepieces/database.sqlite
sqlite> select * from user;
sqlite> update user set password='xxxxx' where email='foo@example.com';
```
Then, I successfully logged in with my old account and flows of it still existed.

Unfortunately, the above method doesn't work for my Ubuntu Linux X86_64 machine.

**Comment by abuaboud** [comment]
It's related to this change I believe

https://github.com/activepieces/activepieces/issues/4413

**Comment by MrMYHuang** [comment]
> It's related to this change I believe
> 
> #4413

Thank you. If the issue is fixed, I will give it a try.

**Comment by abuaboud** [comment]
I am closing this issue, It will be fixed in the next release (after 0.25.1) It's already merged in main.

**Comment by MrMYHuang** [comment]
Thanks! I successfully upgraded activepieces from 0.20.2 to 0.26.1. The login works well and flows are migrated.
